2-structured-data/test/e2e/BookshelfTestBase.php

```diff
@@ -21,15 +21,20 @@
 use Behat\Mink\Session;
 
 /**
- * Class BookshelfTest
+ * Class BookshelfTestBase
  */
-class BookshelfTest extends \PHPUnit_Framework_TestCase
+abstract class BookshelfTestBase extends \PHPUnit_Framework_TestCase
```

Comment:
This would more appropriately live in `test/lib/`. Also, I'd prefer `BaseTest` for consistency with [google-api-php-client](https://github.com/google/google-api-php-client/blob/master/tests/BaseTest.php)